dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/RegistrationChecks.java

```diff
@@ -0,0 +1,7 @@
+package val31.web;
```

Comment:
Needs a copyright header (copy the one used in Person.java)

Reply:
updated